episodes/rse-context.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Mental health challenges specific to RSEs"
+teaching: 15
+exercises: 5
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- What are the specific features of research software engineering?
+- Which specific mental health challenges the RSE context may bring?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Learn about common mental health challenges in the RSE context.
+- Identify challenges you specifically relate to.
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Workload
+
+RSEs are often involved in complex projects involving multiple stakeholders, and sometimes in multiple projects at once. 
+Because it is not always clear to our collaborators what our work consist on, it can lead to request for more work than can be achieved in a normal day.
+This can be especially true for "behind the scenes" work, such as DevOps or Project Management.
+This very high workload and pressure can ultimately lead to total burn out.
+Importantly, even if it doesn't result in straight-up burn out, the constant pressure and feeling of overwhelm negatively impact the RSEs mental health.
+
+## Context switching
+
+On top of the practical workload, the multitude of projects, and tasks within the project (meeting with the users, project management, etc.) leads to **context switching**.
+Context switching is the fact that, when switching between unrelated tasks or projects, your brain needs time to adjust, be reminded of where you were at, and get started on the new task.
+
+## Working alone or forced socialization
+
+Individuals can have very different needs and preferences in terms of social interaction and socialization.
+When these meets are not met, it can lead to a feeling of distress.
+
+## Expectations to keep technical knowledge up to date
+
+The technological landscape is evolving at a very rapid pace, and RSEs are sometimes expected to stay up to date with these evolutions, on top of their day-to-day workload.
+
+## Job insecurity
+
+Previously mentioned structural issues issues and stressors are compounded by the general **job insecurity** in academia **and** in the tech sector, the intersection of which corresponds to RSE jobs. Permanent position in academia are rare and subject to an intense competition, and the tech has seen massive lay-offs in 2021-2024.
```

Comment:
Better to use hard line breaks to format long paragraphs - otherwise later it may be hard to see changes and resolve conflicts